shared/test/test_properties.rb

```diff
@@ -8,11 +8,11 @@
 
 class PropertiesTest < Minitest::Test
 
-  def test_get_set_delete
-    # The Properties API does not need to share a cookie jar with the Channels API.
-    @channels = Rack::Test::Session.new(Rack::MockSession.new(ChannelsApi, "studio.code.org"))
-    @properties = Rack::Test::Session.new(Rack::MockSession.new(PropertiesApi, "studio.code.org"))
+  def setup
+    init_apis
+  end
 
+  def test_get_set_delete
     create_channel
```

Comment:
Do you expect tests here that don't create a channel?  If not, this and the matching `delete_channel` might be good candidates for setup/teardown.

```ruby
def setup
  init_apis
  create_channel
end

def teardown
  delete_channel
end
```